<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>673</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>等线</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>宝钢环冷机台车轮子速度监测</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/images/Resources/images/WheelMonitor.ico</normaloff>:/images/Resources/images/WheelMonitor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTabWidget" name="centralTabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="monitorTab">
       <attribute name="title">
        <string>监视</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1,1">
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>实时输入</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="realVideoLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>匹配图</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLabel" name="imageMatchesLabel">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,1">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>警报</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="alarmPushButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::NoContextMenu</enum>
               </property>
               <property name="toolTip">
                <string extracomment="reset to green"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="MainWindow.qrc">
                 <normaloff>:/images/Resources/images/gray.png</normaloff>:/images/Resources/images/gray.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>128</width>
                 <height>128</height>
                </size>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="lcdGroupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>报警车轮序号</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLCDNumber" name="lcdNumber">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="lineWidth">
                   <number>3</number>
                  </property>
                  <property name="midLineWidth">
                   <number>1</number>
                  </property>
                  <property name="digitCount">
                   <number>3</number>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::Flat</enum>
                  </property>
                  <property name="intValue" stdset="0">
                   <number>888</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>实时</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="numLabel">
               <property name="text">
                <string>车轮序号</string>
               </property>
               <property name="buddy">
                <cstring>numLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="numLineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>55</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>55</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lastSpeedLabel">
               <property name="text">
                <string>车轮转速(m/min)</string>
               </property>
               <property name="buddy">
                <cstring>lastSpeedLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lastSpeedLineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>55</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>55</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="cartSpeedLabel">
               <property name="text">
                <string>台车速度(m/min)</string>
               </property>
               <property name="buddy">
                <cstring>cartSpeedLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="cartSpeedLineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>55</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>55</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="previewTab">
       <attribute name="title">
        <string>预览</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QWidget" name="previewWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playbackTab">
       <attribute name="title">
        <string>故障回放</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_Main">
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>开始(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_Monitor">
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>监测(&amp;M)</string>
     </property>
     <addaction name="action_Start"/>
     <addaction name="action_Stop"/>
    </widget>
    <addaction name="menu_Monitor"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
    <addaction name="action_Restart"/>
   </widget>
   <widget class="QMenu" name="menu_Setting">
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="action_Property"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_Show_Log"/>
    <addaction name="action_Backup_Log"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <addaction name="menu_Main"/>
   <addaction name="menu_Setting"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <action name="action_Start">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>启动(&amp;A)</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>停止(&amp;P)</string>
   </property>
  </action>
  <action name="action_Property">
   <property name="text">
    <string>属性...</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>关于 &amp;Qt</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="action_Restart">
   <property name="text">
    <string>重启(&amp;R)</string>
   </property>
  </action>
  <action name="action_Show_Log">
   <property name="text">
    <string>显示日志(&amp;L)</string>
   </property>
  </action>
  <action name="action_Backup_Log">
   <property name="text">
    <string>备份日志到(&amp;B)...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>numLineEdit</tabstop>
  <tabstop>lastSpeedLineEdit</tabstop>
  <tabstop>cartSpeedLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
